dev-client/src/screens/LocationScreens/components/soilId/SiteSoilCracksSection.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright © 2024 Technology Matters
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+
+import {useCallback} from 'react';
+import {useTranslation} from 'react-i18next';
+
+import {Button} from 'native-base';
+
+import {Icon} from 'terraso-mobile-client/components/icons/Icon';
+import {
+  Box,
+  Heading,
+  Text,
+} from 'terraso-mobile-client/components/NativeBaseAdapters';
+import {RestrictBySiteRole} from 'terraso-mobile-client/components/RestrictByRole';
+import {SITE_EDITOR_ROLES} from 'terraso-mobile-client/model/permissions/permissions';
+import {useNavigation} from 'terraso-mobile-client/navigation/hooks/useNavigation';
+import {useSelector} from 'terraso-mobile-client/store';
+import {selectSoilData} from 'terraso-mobile-client/store/selectors';
+
+type Props = {siteId: string};
+export const SiteSoilCracksSection = ({siteId}: Props) => {
+  const {t} = useTranslation();
+  const navigation = useNavigation();
+  const soilData = useSelector(selectSoilData(siteId));
+
+  const onAddSoilCracksDataPress = useCallback(() => {
+    navigation.push('SOIL_SURFACE', {siteId});
+  }, [navigation, siteId]);
+
+  return (
+    <>
+      <Heading variant="h6" pt="lg" pb="md">
+        {t('site.soil_id.site_data.cracks.title')}
+      </Heading>
+
+      {soilData.surfaceCracksSelect ? (
+        <Text>
+          {t(`soil.vertical_cracking.value.${soilData.surfaceCracksSelect}`)}
+        </Text>
+      ) : (
+        <></>
```

Comment:
Design nit: The Figma has the spacing consistent regardless of if there's data there or not. This change has them slightly different. Just want to make sure Courtney doesn't mind this?

![Screenshot 2024-12-03 at 4 17 27 PM](https://github.com/user-attachments/assets/55397c9f-18be-4d97-b6a2-2128927850d6) vs ![Screenshot 2024-12-03 at 4 17 46 PM](https://github.com/user-attachments/assets/70bf174b-9be5-4acb-ba6e-c81304b4bad6)